### PR TITLE
Adds new readme and changes name to SqsFacade

### PR DESCRIPTION
**Pull request by nrodriguez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +35,45 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install aws_sqs_faker
+    $ gem install sqs_facade
 
 ## Usage
 
-TODO: Write usage instructions here
+Usage is easy. Simple instantiate the class and by default new will return an object with generated dummy values.
```
Comment (alexcopquin) [inline README.md]: "Simply"

File: README.md [hunk 2]
```diff
@@ -18,11 +35,45 @@ And then execute:
...
+```
+To access the full message as a hash use the ```full_message``` method
+
+```ruby
+SqsFacade.new.full_message
+
+-> {"Type"=>"Notification", "MessageId"=>"e1cbeb58-9b36-4ee4-94ff-d195f2b7fcbf", "TopicArn"=>"arn:aws:sns:us-east-1:e9f8a671-2d08-4fc4-b54b-6109c02da13a:some-sns-topic", "Subject"=>"entityupsert", "Message"=>"{\"something\":\"hey\",\"really\":\"there\",\"cool\":\"lonely girl\"}", "Timestamp"=>"2016-05-19T19:53:01.936Z", "SignatureVersion"=>"1", "Signature"=>"DBHK4iOEfwT99DLhLz2EqQ==", "SigningCertURL"=>"https://sns.us-east-1.amazonaws.com/0fae4284696382f2784a1670d2a56b81.pem", "UnsubscribeURL"=>"https://sns.us-east-1.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-east-1:e9f8a671-2d08-4fc4-b54b-6109c02da13a:some-sns-topic:1234"}
+```
+
+You can override certain values by passing it in during instantiation:
```
Comment (alexcopquin) [inline README.md]: "passing them"

File: README.md [hunk 3]
```diff
@@ -32,10 +83,9 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/aws_sqs_faker. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](contributor-covenant.org) code of conduct.
+Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/sqs_facade. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](contributor-covenant.org) code of conduct.
```
Comment (alexcopquin) [inline README.md]: "...welcoming space for collaboration. Contributors are expected to adhere to the ...."

**Comment by alexcopquin** [comment]
I made some anal obsessive compulsive syntax corrections. 
This thing is pretty cool. I feel like a dumbass that we didn't actually use it previously.
One thing, Façade seems to convey another meaning, relating it to the design pattern of the same name. But it does sound catchy.

Hope it helps

**Comment by nrodriguez** [comment]
@alexcopquin Yea I thought about that, but I saw the numerous repos with the name "Facade" so i just figured I'd go with it haha

**Pull request merged**